Assignment1/src/test/java/files/Data.java

```diff
@@ -1,54 +1,65 @@
 package files;
 
 public class Data {
-	
+
 	private int id;
 	private String email;
 	private String first_name;
 	private String last_name;
 	private String avatar;
 	private String password;
-	
+
 	public int getId() {
 		return id;
 	}
+
 	public void setId(int id) {
 		this.id = id;
 	}
+
 	public String getEmail() {
 		return email;
 	}
+
 	public void setEmail(String email) {
 		this.email = email;
 	}
+
 	public String getFirst_name() {
```

Comment:
This needs to be camelCased. Use decorators to tell the json library which field to make this function to. I've left this comment before too. Please review all getter/setter functions and make sure the dont have `_` in their names.

Reply:
changed the field names to camelCased using @JsonProperty("first_name") private String firstName;
removed _ from getters and Setter as well